### DISCUSSION TITLE
Pressing Cmd-ScrollLock puts monitors into mirror mode

### DISCUSSION THREAD

**Discussion by benze** [MonitorControl/MonitorControl#1640] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I have 2 external monitors connected to my M1 Macbook Pro as extended desktop. 

When I run MonitorControl 4.2.0 and press Cmd-ScrollLock on my external keyboard, my monitors switch into Mirror'ed display mode.  So I end up with 2 monitors which mirror the primary screen.  To get out of mirror mode, Cmd-ScrollLock will put it back in Extended Desktop mode.

I've looked through configuration options to see if I can disable this somehow, but do not see anything.

1) Is this a setting that is configured by MonitorControl
2) Is there a way to disable/remap this key configuration

I am not convinced that it is MonitorControl which is causing the displays to switch to Mirror mode, but if I quit MonitorControl, then pressing Cmd-ScrollLock does nothing.

Any suggestions would be appreciated.

Thanks,

**Comment by waydabber** [comment]
Interesting. Normally Command+F1 is used to toggle mirror on the Mac. It might be that somehow this PC keyboard maps that to this? I'll flag it as a bug if it is indeed MonitorControl specific.

**Comment by benze** [comment]
@waydabber I haven't been able to find any other mappings that are defined elsewhere.  My inclination is to question if MonitorControl is somehow intercepting this keyboard shortcut and misinterpreting it as something else?

I'm not convinced that it is a MonitorControl issue, other than quitting MC stops this issue.  And restarting MC reactivates it again.  So it seems to somehow be related.

I've tried using Karabiner - EventViewer to double check the key combination, but the Windows-ScrLk is captured as Cmd-ScrollLock (see below)

```
[
  {
    "type": "down",
    "name": {"key_code":"left_command"},
    "usagePage": "7 (0x0007)",
    "usage": "227 (0x00e3)",
    "misc": "flags left_command"
  },
  {
    "type": "down",
    "name": {"key_code":"scroll_lock"},
    "usagePage": "7 (0x0007)",
    "usage": "71 (0x0047)",
    "misc": "flags left_command"
  },
  {
    "type": "up",
    "name": {"key_code":"scroll_lock"},
    "usagePage": "7 (0x0007)",
    "usage": "71 (0x0047)",
    "misc": "flags left_command"
  },
  {
    "type": "up",
    "name": {"key_code":"left_command"},
    "usagePage": "7 (0x0007)",
    "usage": "227 (0x00e3)",
    "misc": ""
  }
]
```

Let me know if I can provide any additional information.
Thanks,

Eric

**Comment by waydabber** [comment]
This might be a MonitorControl issue indeed. MonitorControl uses https://github.com/nhurden/MediaKeyTap to capture native media keys - so that's where the problem probably lies.
